helpers/github.ts

```diff
@@ -251,28 +252,74 @@ export async function getProjectUrls() {
   return projectUrls;
 }
 
-// Function to calculate total rewards from open issues
-export async function calculateTotalRewards(issues: GitHubIssue[]) {
-  let totalRewards = 0;
+// Function to calculate total rewards and tasks statistics
+export async function calculateStatistics(issues: GitHubIssue[]) {
+  const rewards = {
+    notAssigned: 0,
+    assigned: 0,
+    completed: 0,
+    total: 0,
+  };
+
+  const tasks = {
+    notAssigned: 0,
+    assigned: 0,
+    completed: 0,
+    total: 0,
+  };
+
   await issues.forEach((issue) => {
     const labels = issue.labels as GitHubLabel[];
-    if (issue.state === "open" && labels.some((label) => label.name as string)) {
+    const isAssigned = labels.find((label) => (label.name as string).includes(LABELS.UNAVAILABLE));
+    const isCompleted = issue.state === "closed";
+
+    // Increment tasks statistics
+    tasks.total++;
+    if (isAssigned) {
+      tasks.assigned++;
+    } else {
+      tasks.notAssigned++;
+    }
+
+    if (labels.some((label) => label.name as string)) {
       const priceLabel = labels.find((label) => (label.name as string).includes("Pricing"));
       if (priceLabel) {
+        // ignore pricing not set
+        if (priceLabel.name === "Pricing: not set") return;
+
         const price = parseInt((priceLabel.name as string).split(":")[1].trim(), 10);
-        totalRewards += price;
+
+        // Increment rewards statistics, if it is assigned but not completed
+        if (isAssigned && !isCompleted) {
+          rewards.assigned += price;
```

Comment:
Does this have proper error handling for NaN etc?

Reply:
I've added an error handling now, although i had a check for price not set earlier but NaN check would make it better